website/content/en/docs/upgrading-sdk-version/backport-policy.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Backport Policy
+title: Backport for Operator SDK
```

Comment:
We're already on the SDK site. I don't think we need "for Operator SDK" in the page titles.

I could be convinced otherwise if there's a good reason for this though.

Reply:
Ah I see it aligns with other titles. So :man_shrugging:

I would prefer without the "for Operator SDK" but if most other things use that, its better to be consistent.

Reply:
The idea was that. See `Backport for Operator SDK`. 
Then note that what appears on the menu is the `linkTitle:`. In the past, we change all docs to have for Operator SDK in order to them starts to be returned in the google search.

Reply:
Perhaps we should remove the `for Operator SDK` throughout the site instead as to not be redundant.

Reply:
> In the past, we change all docs to have for Operator SDK in order to them starts to be returned in the google search.

That's right! I had forgotten about that. At that point all of the docs were hosted just right in GitHub. Now that we're on our own domain, that may no longer be an issue. I'm not an SEO expert though.

> Perhaps we should remove the for Operator SDK throughout the site instead as to not be redundant.

If that's the route we go, I'm good with doing it as a follow-on. Whatever we do, let's try to minimize the number of times we break links. That's always a no-fun surprise when CI breaks without warning. And I'm sure it annoys users.

Reply:
So maybe for this PR, let's not break any links related to titles containing "for Operator SDK" or not. And focus just on getting the migration docs renamed/reweighted.

Reply:
Hi @estroz and @joelanford, 

I would like to be sure that we are all in the same page before removing the  `for Operator SDK throughout the site instead as to not be redundant.`

### Is the `for Operator SDK`  showing in the menu? 

No. The menu shows the `linkTitle:`

<img width="245" alt="Screen Shot 2020-09-21 at 17 50 14" src="https://user-images.githubusercontent.com/7708031/93796568-ed6b0680-fc32-11ea-9998-c4877b077326.png">

### Why the SDK docs have `Operator-sdk` in the titles? 

Its motivations came from : https://github.com/operator-framework/operator-sdk/issues/1905
It was added for the docs be properly indexed in the searches.

So, should we keep the titles with Operator-sdk for the docs appears in the searches since people will be looking for something such as; "how to upgrade a version of an operator-sdk project"? , what is the backport policy for operator-sdk? and etc or should we remove it from all docs?

Reply:
Raise an issue to keep all following the same standard after we reach a consensus. See: https://github.com/operator-framework/operator-sdk/issues/3927

Reply:
Closing as solved.